### ISSUE TITLE
Decimal places for Amplitude and Gain

### ISSUE THREAD

**Issue by Bumblebee001** [GrandOrgue/grandorgue#1627]
Is it really necessary to have 6 decimal places for Amplitude and gain?

**Comment by Bumblebee001** [comment]
This goes also for pitch tuning and possibly others as well.

**Comment by oleg68** [comment]
@larspalo @rousseldenis would one decimal place be enough?

**Comment by larspalo** [comment]
@oleg68 With AmplitudeLevel and Gain I've basically ever only used integers manually. With tuning I might have occasionally used one decimal when tuning manually. With embedded pitch from LoopAuditioneer I currently have 2 decimals for the pitch fraction. But for a planned update of LoopAuditioneer I'm even considering going with just whole numbers for the embedded pitch info (the cent value equivalent of pitch fraction) as that's what the recent stock libsndfile uses nowadays (earlier it even wasn't implemented at all, and that's why I did a separate statically linked custom build of it) which would enable dynamic linking and hopefully less maintenance burden.

For the MinVelocityVolume of tracker actions I've used 0.001 as value in some odfs, but I think that's the highest precision I've ever used for any float/double manually.

As the Organ Settings dialog were created when wxSpinCtrlDouble didn't exist (that's why there is a separate wxTextCtrl linked together with a wxSpinCtrl) the current layout could be further simplified. (it's actually quite easy to control the number of decimal places and even separate increment value of a wxSpinCtrlDouble)

So to conclude, I really don't think we'd loose anything significant if we limit the number of (shown and from the gui possible to set) decimals to just one (0.0).
